backend/app2/api/routes/sounds.py

```
@@ -188,6 +190,32 @@ async def create_drum_track(
         )
 
 
+@router.get("/drum", response_model=Page[DrumTrackRead])
+async def get_drum_tracks(
+    current_user: Dict[str, Any] = Depends(get_current_user),
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)